jwt/jwt_test.go

```diff
@@ -88,3 +88,9 @@ func TestVerify_BadClaims(t *testing.T) {
 	assert.Error(t, err)
 	assert.Equal(t, jwt.Decoded{}, decoded)
 }
+
+func Test_Decode_Empty(t *testing.T) {
+	decoded, err := jwt.Decode("")
```

Comment:
at some point we should change to return nil instead of an empty struct, but that's beyond the scope of this pr.

Reply:
I had the same thought... ticket! https://github.com/TBD54566975/web5-go/issues/48

Reply:
If we must 😭 would like to be consistent about it tho!